docs/conf/portland/2021/writing-day.rst

```diff
@@ -64,9 +64,11 @@ For more information, see our `GitLab issue <https://gitlab.com/gitlab-org/techn
 
 When you find an issue you want to work on:
 
-#. Inform the GitLab team that you're working on an issue.
-   Add a comment that mentions `@gl-docsteam` (or tell us in person!).
 #. Create a merge request with your proposed changes.
+#. Add labels to the merge request. Use the same ones assigned to your issue, and add `community contribution` as well.
```

Comment:
@selhorn if you're not a member of the project, I don't think you can actually add labels.

Reply:
Fixing in https://github.com/writethedocs/www/pull/1496